tests/lib/rules/no-confusing-arrow.js

```diff
@@ -23,45 +23,47 @@ ruleTester.run("no-confusing-arrow", rule, {
         "a => { return 1 ? 2 : 3; }",
         "var x = a => { return 1 ? 2 : 3; }",
         "var x = (a) => { return 1 ? 2 : 3; }",
-        { code: "var x = a => (1 ? 2 : 3)", options: [{ allowParens: true }] }
+        { code: "var x = a => (1 ? 2 : 3)", options: [{ allowParens: true }] },
+        "var x = a => (1 ? 2 : 3)"
```

Comment:
Would you add tests for `allowParens: false`?
Looks there are no tests for the option...

Reply:
Actually there are. (in invalid tests)
I think there are no **valid tests** for the option, just as all the cases will report an error. 😄

Reply:
Originately, were the line 23-35 valid tests for `{allowParens:false}`?

Reply:
oops, I missed it, will add it later.